From 3400ee1e9385ad3ef4f3a3f05be780759e262735 Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Mon, 10 Jan 2022 11:00:22 +0100
Subject: [PATCH] style: Fix the iconless category tile min width

This substracts the tile's border width to its minimum width to make it
consistent with the other category tiles.
---
 src/gtk-style.css | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gtk-style.css b/src/gtk-style.css
index 2f61b38be..5e30caced 100644
--- a/src/gtk-style.css
+++ b/src/gtk-style.css
@@ -204,6 +204,8 @@ clamp.large .category-tile:not(.category-tile-iconless) {
 
 .category-tile.category-tile-iconless {
 	font-weight: normal;
+	/* We have to remove the border: 160px - 2*1px = 158px */
+	min-width: 158px;
 	padding: 0;
 	background-image: none;
 }
-- 
2.30.2

